### PR TITLE
docs: added value param wrt. method signature

### PR DESCRIPTION
**Pull request by mumairofficial**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Added missing value param for `append` method documentation. This change will help IDEs to show better documentation while code completion.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by mumairofficial** [comment]
@googlebot I fixed it.

**Comment by mumairofficial** [comment]
@googlebot I fixed it.

**Comment by kapunahelewong** [comment]
Thanks for this, @mumairofficial. My apologies for the delay. Requesting @jbogarthyde for review as she knows the API docs well. @jbogarthyde, will you let me know when you're ready and I'll come back to help get this through the rest of the way? Thanks!

**Review by jbogarthyde — APPROVED** [review]
Good catch, thanks!

**Comment by kapunahelewong** [comment]
Thanks, @jbogarthyde! @mumairofficial, all you need to do now is [fix the conflicts](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line) and rebase since it is a bit behind master. Here are my [own notes for rebasing](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) if you'd like them. Will you tag me when you're ready? Thanks!

File: packages/common/http/src/headers.ts [hunk 1]
```diff
@@ -140,6 +140,7 @@ export class HttpHeaders {
    * header values.
    *
    * @param name The header name for which to append the values.
+   * @param value The value to set or overide for a given name.
```
Comment (trotyl) [inline packages/common/http/src/headers.ts]: The value here is to append, not set or override.

Comment (jbogarthyde) [reply packages/common/http/src/headers.ts]: Oops, you’re right!  How about simply “The value to append.”

Comment (jbogarthyde) [reply packages/common/http/src/headers.ts]: Oops, you’re right! How about simply “The value to append.”

**Review by jbogarthyde — CHANGES_REQUESTED** [review]

File: packages/common/http/src/headers.ts (hunk 1 above)
Comment (trotyl) [reply packages/common/http/src/headers.ts]: @jbogarthyde SGTM

Comment (mumairofficial) [reply packages/common/http/src/headers.ts]: I have updated ~set~ to append

**Review by jbogarthyde — COMMENTED** [review]
No, it does not override, it just appends. Please just use the shorter phrase: "The value to append."

**Comment by mumairofficial** [comment]
@jbogarthyde I have removed the override statement as you suggested

**Review by jbogarthyde — APPROVED** [review]
Thanks. This works. I think you have to rebase again.

**Comment by jbogarthyde** [comment]
@mumairofficial 
Looks like the rebase failed - try again?

**Comment by mumairofficial** [comment]
@jbogarthyde sure I am trying again

**Comment by mumairofficial** [comment]
@googlebot I fixed it.

**Comment by mumairofficial** [comment]
@googlebot I consent.

**Comment by kapunahelewong** [comment]
Hey @mumairofficial, it looks like the result of that last step was unintentional. To undo it, you should be able to do a `git reflog` and find the place you want to back up to. Checkout the accepted answer on this [StackOverflow question](https://stackoverflow.com/questions/11722533/rollback-a-git-merge/29110174). First, make sure to back up your work (you should just be able to copy and paste it somewhere else on your computer separate from your project because doing a git reset will erase things). If it's still not working out, please let me know.

**Comment by mumairofficial** [comment]
@kapunahelewong Sure I can try.

**Comment by kapunahelewong** [comment]
Hey, @mumairofficial, I think I fixed it? Mind if I push up what I did? Honestly it was a little tricky to come to my eventual (I think) solution, and I'm not sure of the answer to your question.

**Comment by kapunahelewong** [comment]
I'll have to force push probably so you may want to back up your work.

**Comment by mumairofficial** [comment]
@kapunahelewong yes you may push your solution. Also I am facing issues with my git behaving very strange. Thanks for backup suggestion to safe life.

**Comment by kapunahelewong** [comment]
Gasp! It worked!!! Is all your work still in there though? If not, keep your back up, then pull this down and edit it. If your git is still acting funny, just put it in a comment and I'll add it for you. Easy peasy. :)

**Comment by mumairofficial** [comment]
@kapunahelewong Yup recovered my work. Learned new git lesson :) Ends up replacing .git directory

**Comment by jbogarthyde** [comment]
Caretaker: This is a small, doc-only change that shouldn't actually need code-owner approval or G3.

**Pull request closed without merging**